modules/effects/arrow-effects-data/src/test/kotlin/arrow/effects/predef.kt

```diff
@@ -0,0 +1,14 @@
+package arrow.effects
+
+import arrow.Kind
+import arrow.core.extensions.either.eq.eq
+import arrow.core.extensions.option.eq.eq
+import arrow.effects.typeclasses.Duration
+import arrow.effects.typeclasses.seconds
+import arrow.typeclasses.Eq
+
+fun <A> EQ(EQA: Eq<A> = Eq.any(), timeout: Duration = 60.seconds): Eq<Kind<ForIO, A>> = Eq { a, b ->
```

Comment:
We have at least 2 uses of this that I've seen during the review. We should replace those too in another diff.